bin/clever.js

```diff
@@ -87,6 +88,30 @@ function run() {
   var addonNameArgument = cliparse.argument("addon-name", { description: "Addon name" });
   var addonProviderArgument = cliparse.argument("addon-provider", { description: "Addon provider" });
 
+  //PARSERS
+  var flavorParser = function(flavor) {
+    var flavors = Application("listAvailableFlavors")();
+
+    if(flavors["words"].indexOf(flavor) == -1) {
```

Comment:
instead of using the autocomplete data structure, can you make Application.listAvailableFlavors return a simple list, and wrap it in words in the autocomplete handler?